### PR TITLE
data: mhdamaan79

### PR DESCRIPTION
**Pull request by mhdamaan79**
## Fixes Issue

Adding a profile to the LinkFree account

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

Thanks for letting me create a profile on LinkFree!

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4010"></a>

### CODE REVIEW

File: data/mhdamaan79.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+  "name": "Mohd Amaan",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Computer Science Student 👨‍💻 | Learning Java + DSA | Learning MERN Stack Web Development 🌐 | Bachelor of Computer Applications (BCA) | Tech & Open Source Enthusiast 🚀",
+  "avatar": "https://github.com/mhdamaan79.png"
```
Comment (stevenovak123) [inline data/mhdamaan79.json:6]: a comma is required after avatar `,`

Comment (stevenovak123) [inline data/mhdamaan79.json:6]: Comma required at the end of line

**Review by stevenovak123 — CHANGES_REQUESTED** [review]

**Comment by mhdamaan79** [comment]
@stevenovak123 thanks, I've just corrected it.

File: data/mhdamaan79.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/mhdamaan79" },
+    { "icon": "FaGithub", "url": "https://github.com/mhdamaan79" },
+    { "icon": "FaLinkedin", "url": "https://www.linkedin.com/in/mhdamaan79/" }
+  ],
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/mhdamaan79",
+      "icon": "FaTwitter",
```
Comment (stevenovak123) [inline data/mhdamaan79.json:23]: here a comma is not required as its the end. Please correct. 
`  "icon": "FaTwitter"`

**Review by stevenovak123 — CHANGES_REQUESTED** [review]

**Review by stevenovak123 — APPROVED** [review]
LGTM, Thank you for your contribution

**Review by money8203 — APPROVED** [review]

**Comment by mhdamaan79** [comment]
@stevenovak123 and @money8203 any update?
I mean what's next, is there anything left to be changed...

**Comment by stevenovak123** [comment]
Wait till a maintainer merges your request. thats it

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**